build.gradle

```diff
@@ -132,9 +129,9 @@ uploadArchives {
             // when we're ready to publish to Maven Central, use this repository instead
             // (make sure a project has been created for us first!)
             // see: http://central.sonatype.org/pages/ossrh-guide.html#create-a-ticket-with-sonatype
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: project.property("sonatypeUsername"), password: project.property("sonatypePassword"))
-            }
+            //repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
```

Comment:
but we do need to push it to our artifactory. what I do is make these changes in a separate branch, and execute a build from it (the other branch) once the code chages are merged